<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="Serial_Port">
      <property name="text">
       <string>Serial Port:</string>
      </property>
      <property name="buddy">
       <cstring>Port</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="Port">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <item>
       <property name="text">
        <string>ttyUSB0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyUSB1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyUSB2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyUSB3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyUSB4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyUSB5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyUSB6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ttyUSB7</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Baud_Rate_2">
      <property name="text">
       <string>Baud Rate:</string>
      </property>
      <property name="buddy">
       <cstring>Baud_Rate</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="Baud_Rate">
      <property name="currentText">
       <string>115200</string>
      </property>
      <property name="currentIndex">
       <number>11</number>
      </property>
      <item>
       <property name="text">
        <string>110</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>300</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>256000</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="parityLabel">
      <property name="text">
       <string>Parity:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="dataBitsBox">
      <item>
       <property name="text">
        <string>NoParity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EvenParity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OddParity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SpaceParity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MarkParity</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="stopBitsLabel">
      <property name="text">
       <string>Stop bits:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="parityBox">
      <item>
       <property name="text">
        <string>OneStop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OneAndHalfStop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TwoStop</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dataBitsLabel">
      <property name="text">
       <string>Data bits:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="flowControlBox">
      <property name="currentText">
       <string>Data8</string>
      </property>
      <item>
       <property name="text">
        <string>Data5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Data6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Data7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Data8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="flowControlLabel">
      <property name="text">
       <string>Flow control:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="stopBitsBox">
      <item>
       <property name="text">
        <string>NoFlowControl</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HardwareControl</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SoftwareControl</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
